images/shared-images-controller/main.go

```diff
@@ -79,15 +79,21 @@ func pullRequiredImages(ctx context.Context, tag string) error {
 
 	for _, version := range versions {
 		log.Infoln("Kubevirt Provider version: ", version)
+
 		name := fmt.Sprintf("%sk8s-%s:%s", kubevirtci_repo, version, tag)
+		// k8s-1.26 kubevirtci provider was renamed to k8s-1.26-centos9
+		// to track the change to CentOS Stream 9
+		if version == "1.26" {
+			name = fmt.Sprintf("%sk8s-%s-centos9:%s", kubevirtci_repo, version, tag)
+		}
 		if _, exists := imageNames[name]; exists {
 			log.Infoln("Image already present:", name)
 			continue
 		}
 		log.Infoln("Pulling image: ", name)
 		_, err := images.Pull(ctx, name, nil)
 		if err != nil {
-			return err
+			log.WithError(err).Errorf("Failed to pull image '%s'", name)
```

Comment:
Let's aggregate the errors and let the caller log it as we do now.

Reply:
@xpivarc - I was hoping to just log the error when it happens and move on to pull the next image - why would I aggregate the errors? If it fails to pull an image - we still want it to try to pull the rest.

Reply:
Ok, fair enough. I thought we might be doing some logic based on returned error. We are not doing it.